### ISSUE TITLE
visual selection with mouse doesn't include last character

### ISSUE THREAD

**Issue by kevintighe** [jaredpar/VsVim#2366]
The behavior of selecting text with the mouse in VsVim 2.6.1189 differs a bit from gVim 8.1.1.  

In gVim, when highlighting some text with the mouse, the last character that the cursor hovers over is included in the selection.  In VsVim, the last character is not included.  

Here is an example of what I mean (selection, yank, put):

**gVim:**
![gvim2](https://user-images.githubusercontent.com/6350719/45429375-3285f680-b671-11e8-978b-46f7fd8c6f78.gif)

**VsVim:**
![vsvim2](https://user-images.githubusercontent.com/6350719/45429384-3580e700-b671-11e8-9077-6a5e2c1e03bc.gif)

Thanks!

**Comment by ricksladkey** [comment]
The former is enabled by the setting `:set selection=inclusive` and the latter is enabled by the setting `:set selection=exclusive`. You can see the current value of the selection setting is by typing `:set selection?`.

The default settings are rather complicated with both gvim and VsVim, however with both there two fundamentally different types of selection that both support: xterm-style and Windows-style.

Specifically both gvim and VsVim support the commands `:behave xterm` (which sets `selection=inclusive`) and `:behave mswin` (which sets `selection=exclusive`).

The best advice is to chose whether you prefer xterm-style or Window-style and then put the corresponding `:behave` command in both your gvim rc file and your VsVim rc file.

If this clarifies the differences you are seeing, then please close this issue.

**Comment by kevintighe** [comment]
@ricksladkey Thanks for the quick response!

I confirmed that both VsVim and gVim are using `inclusive` selection via `:set selection?`.  I then tried using `:behave xterm` in both, which results in the same differing behavior as above.  In VsVim, the character under the block cursor is not included in the selection, while in gVim it is.

I also tried `exclusive` via both `:set selection=exclusive` and `:behave mswin`.  In both cases the behavior of VsVim and gVim matched exactly.  It's only with `inclusive` that I see the difference.  

At this point I'd be happy to just switch to `exclusive`...but the Vim plugin I'm using in VSCode doesn't support that ;).
